### PR TITLE
[#267] Add PHP Security Checker GitHub Action

### PR DESCRIPTION
**Pull request by pcs289**

### CODE REVIEW

File: .github/workflows/php-security.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  php-security-checker:
+    name: PHP Security Checker
+    runs-on: ${{ matrix.operating-system }}
+
+    strategy:
+      matrix:
+        dependencies:
+          - "locked"
+        php-version:
+          - "7.4"
```
Comment (kpicaza) [inline .github/workflows/php-security.yml:18]: Only for testing, we can try different dependencies for security
```suggestion
      matrix:
        dependencies:
          - "lowest"
          - "highest"
          - "locked"
        php-version:
          - "7.4"
          - "8.0"
```

**Comment by kpicaza** [comment]
Try executing `vendor/bin/monorepo-builder validate` to check inconsistencies between global versions and package versions, good job ;-D

**Comment by xserrat** [comment]
Looks really good!! Great job :)

**Review by kpicaza — APPROVED** [review]
:ship::ship::ship:

**Pull request merged**